styles.css

```diff
@@ -62,6 +73,149 @@ h1 {
 
 .social-links {
   display: flex;
-  list-style: none;
-  padding: 0;
+}
+
+.work {
+  background: #f3f3f3;
+  padding: 114px 24px 0 24px;
+  display: grid;
+  grid-gap: 80px;
+}
+
+.card {
+  background: white;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  padding: 16px;
+}
+
+.card-img {
+  width: 100%;
+}
+
+.card-title {
+  margin-top: 12px;
+  font-size: 32px;
+  font-weight: 700;
+  margin-bottom: 20px;
+}
+
+.card-details {
+  padding-bottom: 8px;
+}
+
+.card-details * {
+  margin-right: 12px;
+  color: #7a869a;
+  font-weight: 600;
+  font-size: 14px;
+}
+
+.active-span {
+  color: #344563;
+}
+
+.card p {
+  font-size: 16px;
+  color: #344563;
+  line-height: 24px;
+  margin-bottom: 0;
+}
+
+.card ul {
+  margin-top: 12px;
+  display: flex;
+  margin-bottom: 20px;
+}
+
+.card li {
+  background: #ebebff;
+  color: #6070ff;
+  font-size: 12px;
+  font-weight: 500;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 4px 12px;
+  border-radius: 8px;
+  margin-right: 8px;
+}
+
+.primary-btn {
+  color: #396df2;
+  font-size: 17px;
+  font-weight: 500;
+  background: white;
+  border: 1px solid #396df2;
+  border-radius: 8px;
+  padding: 12px 13px;
+  margin-bottom: 12px;
+  font-family: "Poppins", Arial, Helvetica, sans-serif;
+}
```

Comment:
- [ ] You did a great ajob in styling the `Get resume` button however, please make to implement the button interactions design `Active`, `Hover` and `Pressed` `Disabled`.
  
    - Kindly, take a loot to your Figma design and implement the required button interactions
       
    ![image](https://user-images.githubusercontent.com/87197412/185106473-cc11861d-36b3-4555-8a97-8ebc8d15b9d2.png)